internal/playlist_creator/config/ConfigLoader.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+)
+
+type Config struct {
+	InputPath          string   `json:"music folder"`
+	OutputPath         string   `json:"output folder"`
+	PlaylistName       string   `json:"playlist name"`
+	ExtensionWhitelist []string `json:"Limit extensions to"`
+	ShufflePlaylist    bool     `json:"shuffle playlist"`
+	SplitPlaylist      bool     `json:"split playlist"`
+	ChunkSize          int      `json:"split playlist into chunks of"`
+	ListExtensions     bool     `json:"print a list of extensions"`
+	ListFiles          bool     `json:"print a list of file names"`
+	ListLimit          int      `json:"limit list file names to"`
+	ReadTags           bool     `json:"read tags"`
+}
+
+func LoadConfig() (*Config, error) {
+	return loadConfigFromFile("config/config.json")
```

Comment:
I don't think you are bundling the `config/config.json` with the binary. This would fail if you try to run the binary away from this repository 😄 

 you can use the `embed` package to add the file.
 Another option is to use configuration packages like [Viper](github.com/spf13/viper)